inst/extdata/howto_datainput.md

```diff
@@ -0,0 +1,5 @@
+- `shinyFuse` accepts **tab-separated text files**, as commonly produced by the workflow of `annoFuse`
+- the file should have a **header row**, where the column names are specified
+- if your data in stored in Excel sheets, export them to tsv format for compatibility with `shinyFuse`
+
+Here below you can find a screenshot of the first rows for a file which will be correctly loaded by `shinyFuse`, such as the `PutativeDriverAnnoFuse.tsv` provided bundled with the package.
```

Comment:
```suggestion
Below is a screenshot of for a minimal file which will be correctly loaded by `shinyFuse`.
To take full advantage of all features, we recommend running fusion analysis, annotation, followed by `annoFuse` as noted [here](https://github.com/d3b-center/annoFuse#install-package).
For a fully annotated input file example, see the [`PutativeDriverAnnoFuse.tsv`](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/PutativeDriverAnnoFuse.tsv) provided bundled with the package.
```

Note: that file link will be broken until we merge with master, unless @kgaonkar6, you want to push the file to master right now (I am ok with that), while keeping putative oncogenic until exemplary is merged.

Reply:
Please add one of these images as the example (the users who will need this on the shiny server will likely be non-computational and I think the current image is hard to discern for that audience). Also, this image highlights some oncogenic fusions.

<img width="637" alt="Screen Shot 2020-09-04 at 11 50 20 AM" src="https://user-images.githubusercontent.com/207034/92258582-f6bf4980-eea4-11ea-8e6b-09618f6419c5.png">

<img width="625" alt="Screen Shot 2020-09-04 at 11 53 40 AM" src="https://user-images.githubusercontent.com/207034/92259573-4d2c8800-eea5-11ea-9e5b-d5e9fcf605c1.png">

<img width="623" alt="Screen Shot 2020-09-04 at 11 52 46 AM" src="https://user-images.githubusercontent.com/207034/92259653-5289d280-eea5-11ea-9b5a-bbf70248edb6.png">

Reply:
I like this examples with minimal sets of columns. 
And of the three, I like most the solarized one (number 3) - happy to replace the current one with this!